app/Http/Controllers/Bank/NordigenController.php

```diff
@@ -0,0 +1,304 @@
+
+            if (!$existing_bank_integration) {
+
+                $bank_integration = new BankIntegration();
+                $bank_integration->integration_type = BankIntegration::INTEGRATION_TYPE_NORDIGEN;
+                $bank_integration->company_id = $company->id;
+                $bank_integration->account_id = $company->account_id;
+                $bank_integration->user_id = $company->owner()->id;
+                $bank_integration->nordigen_account_id = $nordigen_account['id'];
+                $bank_integration->bank_account_type = $nordigen_account['account_type'];
+                $bank_integration->bank_account_name = $nordigen_account['account_name'];
+                $bank_integration->bank_account_status = $nordigen_account['account_status'];
+                $bank_integration->bank_account_number = $nordigen_account['account_number'];
+                $bank_integration->nordigen_institution_id = $nordigen_account['provider_id'];
+                $bank_integration->provider_name = $nordigen_account['provider_name'];
+                $bank_integration->nickname = $nordigen_account['nickname'];
+                $bank_integration->balance = $nordigen_account['current_balance'];
+                $bank_integration->currency = $nordigen_account['account_currency'];
+                $bank_integration->disabled_upstream = false;
+                $bank_integration->auto_sync = true;
+                $bank_integration->from_date = now()->subDays(90); // default max-fetch interval of nordigen is 90 days
+
+                $bank_integration->save();
+
+                array_push($bank_integration_ids, $bank_integration->id);
+
+            } else {
+
+                // resetting metadata for account status
+                $existing_bank_integration->balance = $account['current_balance'];
+                $existing_bank_integration->bank_account_status = $account['account_status'];
+                $existing_bank_integration->disabled_upstream = false;
+                $existing_bank_integration->auto_sync = true;
+                $existing_bank_integration->from_date = now()->subDays(90); // default max-fetch interval of nordigen is 90 days
+                $existing_bank_integration->deleted_at = null;
+
+                $existing_bank_integration->save();
+
+                array_push($bank_integration_ids, $existing_bank_integration->id);
+            }
+
+        }
+
+        // perform update in background
+        $company->account->bank_integrations->where("integration_type", BankIntegration::INTEGRATION_TYPE_NORDIGEN)->where('auto_sync', true)->each(function ($bank_integration) {
+            ProcessBankTransactionsNordigen::dispatch($bank_integration);
+        });
+
+        // prevent rerun of this method with same ref
+        Cache::delete($data["ref"]);
+
+        // Successfull Response => Redirect
```

Comment:
Use this here:

        return isset($context["is_react"]) && $context['is_react'] ? redirect(config('ninja.react_url'). "/#/settings/bank_accounts") :  redirect(config('ninja.app_url'));

Reply:
i changed the the implementation at the RequestLevel for default behavior, when this prop is not provided, to match default redirect in the react/default application